### ISSUE TITLE
`no-inline-function-event-listener` false-positive with `once`

### ISSUE THREAD

**Issue by boris-petrov** [foad/eslint-plugin-listeners#3]
I believe doing:

```js
element.addEventListener('click', () => console.log('clicked'), { once: true });
```

Is OK because of `once: true` so perhaps the rule should ignore these cases?

**Comment by foad** [comment]
Good point, fixed in v1.2.0, thanks for raising
